janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/AbstractIndexStatusWatcher.java

```diff
@@ -85,5 +91,13 @@ public S pollInterval(long poll, TemporalUnit pollUnit) {
         this.poll = Duration.of(poll, pollUnit);
         return self();
     }
+    
+    boolean hasStatuses(SchemaStatus status) {
+        return !statuses.stream().filter(s -> s.equals(status)).collect(Collectors.toList()).isEmpty();
+    }
 
+    String statusesToString() {
```

Comment:
Same as above ... can this method be removed with usages replaced with `statuses.toString()`?